### PR TITLE
Blogpost/artifactory gcp

### PR DESCRIPTION
**Pull request by loicalbertin**
Blog post on artifactory on GCP

Part of ystia/yorc#264

### CODE REVIEW

**Comment by adidanes** [comment]
- Add dot (.) to the end of all the phrases 
- **Orchestrator** ...., **to** design and deploy applications on multiple platforms.
- **Forge** ...,Training Models and Data Sets.
- **Deploy applications...** The phrase could be reformed like this : 
Applications deployed with the Ystia Orchestrator are natively portable across infrastructures as they are compliant with the TOSCA (..) specification.

The Ystia orchestrator is able to deploy applications that are are compliant with the TOSCA (..) specification, and thus  natively portable across infrastructures.

**Comment by loicalbertin** [comment]
> * Add dot (.) to the end of all the phrases
> * **Orchestrator** ...., **to** design and deploy applications on multiple platforms.
> * **Forge** ...,Training Models and Data Sets.
> * **Deploy applications...** The phrase could be reformed like this :
>   Applications deployed with the Ystia Orchestrator are natively portable across infrastructures as they are compliant with the TOSCA (..) specification.

@adidanes I have some difficulties to locate comments. Could you make them inline?

**Review by laurentganne — APPROVED** [review]
Looks good!

File: _posts/2019-01-15-jfrog.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+So, for a year or so we used Bintray to store our CI build artifacts. As Bintray is designed to serve release artifacts we experienced some
+difficulties. After our second time being rate-limited due to disk quota overuse (mainly due to the number docker images we built),
+JFrog guys got in touch with us with good news: they offer [Artifactory Pro](https://jfrog.com/artifactory) accounts hosted on GCP for selected open source projects.
+
+While we knew Artifactory and were convinced that it is one of the best solution around to manage artifacts, we didn't consider it yet for two reasons. First we needed to manage Docker images artifacts, which are not supported on the open source version of Artifactory. And secondly, as an open source community we do not have infrastructures to host such products in a community-open way.
+
+We didn't hear about this program, so JFrog email sounded just like a christmas gift in this 20th of december!
+
+We filled-up [the application form](https://jfrog.com/artifactory/saas-gcp/) and few minutes after doing this an instance was live on GCP.
```
Comment (adidanes) [inline _posts/2019-01-15-jfrog.md:27]:
```suggestion
We filled-up [the application form](https://jfrog.com/artifactory/saas-gcp/) and few minutes after doing this an instance was alive in GCP.
```

File: _posts/2019-01-15-jfrog.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+By looking for a way to store these artifacts we found that JFrog offers [Bintray](https://bintray.com) account free of charge for open source projects.
+Creating a Bintray account is as simple as signing in with GitHub and importing your project or organization from GitHub.
+
+So, for a year or so we used Bintray to store our CI build artifacts. As Bintray is designed to serve release artifacts we experienced some
+difficulties. After our second time being rate-limited due to disk quota overuse (mainly due to the number docker images we built),
+JFrog guys got in touch with us with good news: they offer [Artifactory Pro](https://jfrog.com/artifactory) accounts hosted on GCP for selected open source projects.
+
+While we knew Artifactory and were convinced that it is one of the best solution around to manage artifacts, we didn't consider it yet for two reasons. First we needed to manage Docker images artifacts, which are not supported on the open source version of Artifactory. And secondly, as an open source community we do not have infrastructures to host such products in a community-open way.
```
Comment (adidanes) [inline _posts/2019-01-15-jfrog.md]:
```suggestion
While we knew Artifactory, and were convinced that it is one of the best solutions around to manage artifacts, we didn't consider it yet for two reasons. First we needed to manage Docker image artifacts, which are not supported on the open source version of Artifactory. And secondly, as an open source community we do not have infrastructures to host such products in a community-open way.
```

**Pull request merged**